### PR TITLE
added Italian Translation of notes (amiibo_db_link)

### PR DESCRIPTION
**Pull request by JostenSyon**
Added All zelda related notes for Italian language by using proper note_it, comment on auto-generated content to prevent editing it my mistake

### CODE REVIEW

**Review by solosky — CHANGES_REQUESTED** [review]
Thank your contribution on this!

The PR  looks good for me and I could merge to develop branch ready for next release. 

More and more contributors are offering support in various languages, and I am very grateful for their support. However, at the same time, this multi language support will increase the maintenance cost of this project. For example, adding text for new features requires translation work. 

Unfortunately, I am unable to provide translation and maintenance work for additional languages. I am considering whether to stop accepting more language merge requests in the future because I may not be able to maintain these languages. 

I can only provide language support in Chinese and English.

File: README.md [hunk 1]
```diff
@@ -1,20 +1,24 @@
 # Pixl.js
 
-这是一个基于原版[Pixl.js](http://www.espruino.com/Pixl.js)的复刻版本，主要的功能是用来模拟Amiibo。
+Fork progetto originale [Pixl.js](http://www.espruino.com/Pixl.js)
```
[the remark this answers is not in the document]
Comment (JostenSyon) [reply README.md:3]: really sorry, committed by mistake, right now i'm doing a proper docs /it/ documentation ;)

[the remark this answers is not in the document]
Comment (JostenSyon) [reply README.md:3]: btw, when, and I hope will be not necessary, you will need some space due too many languages, I can contribute to make an EU firmware to give some free space ;) (at least for IT-ES-FR-EN there is no problem for me)

[the remark this answers is not in the document]
Comment (solosky) [reply README.md:3]: if you say "space" means available flash memory, that's not an issue currently, we still have around 30K available flash memory for new features and language files. 

BTW, I am working on new hardware version with more flash memory (1M) and new RC522 reader chip on it, there will be new feature added in firmware, I may need your support to translation text to IT-ES-FR in future..

**Review by solosky — APPROVED** [review]
LGTM

**Comment by solosky** [comment]
Thanks, I will release new firmware version next week.

**Comment by JostenSyon** [comment]
I’m ready for an main-eu firmware 😂

Il giorno mer 3 apr 2024 alle 19:23 MayanKoyote ***@***.***>
ha scritto:

**Pull request merged**